ui/src/message_popup/transformations.jsx

```diff
@@ -20,8 +20,9 @@ export function withIndicator(question) {
   };
 }
 
-export function questionsForIndicator(indicatorId) {
-  const withIndicators = _.compact(allQuestions.map((question) => {
+export function questionsForIndicator(indicatorId, questions) {
+  const questionsList = (questions === undefined) ? allQuestions : questions;
```

Comment:
Since this would change how other non-authoring files gather the questions, I decided I would handle it in a separate pull request where we start using the database questions in the actual message popup experience.